### ISSUE TITLE
Provide haptic feedback

### ISSUE THREAD

**Issue by Ecco** [numworks/epsilon#933]
#### Problem you'd like to fix
It might not be obvious when a key is pressed on a touchscreen

#### Describe the solution you'd like
Some platforms (e.g. Android) allow a small vibration to be emitted on a touch event. That could be a great addition.

**Comment by 0b101** [comment]
What about also adding some sort of shadow (like in the HP Prime Lite app) or image change (like in the Free42 app) when a key is pressed?

**Comment by Ecco** [comment]
> What about also adding some sort of shadow (like in the HP Prime Lite app) or image change (like in the Free42 app) when a key is pressed?

Given the key is beneath the user's finger, I don't think this would even be visible 😄

**Comment by M4xi1m3** [comment]
If this happens, you should make it configurable. Some users like me hate their phone vibrating on each and every key press.

**Comment by Ecco** [comment]
Indeed. That being said there should be a system-wide settings to disable haptic feedback, se we probably don't need to provide our own UI for this.

**Comment by RedGl0w** [comment]
Closed by https://github.com/numworks/epsilon/pull/1633

**Comment by RedGl0w** [comment]
Even more closed with e15

**Comment by txoof** [comment]
The haptics on the android v23.2.1 are a intense! Is there a way to scale this back?

**Comment by adri1numworks** [comment]
Comment from @rbsamoht :
The Android application (version 23.2.1) has some intense haptic feedback, that exceeds the usual level of haptic feedback on other applications (for the built-in Android keyboard for example).

This haptic feedback can be particularly annoying when the phone is sitting on a desk.

### Describe the solution you'd like

Ideally, this option would also allow to disable completely haptic feedback.

### Describe alternatives you've considered
I tried to check all the option related to haptic feedback in Android settings but none of them worked.

It seems to be an application specific setting.

Additional context
I see that this issue was already reported in a few places:

https://github.com/numworks/epsilon/issues/933#issuecomment-2242156287
[a Reddit thread](https://www.reddit.com/r/calculators/comments/1b80ah8/how_do_i_disable_vibration_on_numworks_emulator/)
Many comments in the application reviews in Google Play (below are some quotes from the review section of the Play Store):
La vibration à l'appuie d'une touche est beaucoup trop forte !!

Enlevez les vibrations !!!!

Meilleure marque de calculatrice extrêmement intuitive mais juste dites moi comment enlever les vibrations !!!!!!!!!!

Ce serait 5 étoiles si on pouvait désactiver les vibrations [...]

Appli incroyable, seul bémol : les vibrations qui sont très violentes ! Ce serait vraiment bien de permettre aux utilisateurs de la régler ou de l'éteindre complètement. À part ça, super
